bitnami/airflow/README.md

```diff
@@ -149,6 +155,26 @@ The following tables lists the configurable parameters of the Kafka chart and th
 | `metrics.image.pullPolicy`                | Image pull policy                                                                                    | `IfNotPresent`                                               |
 | `metrics.image.pullSecrets`               | Specify docker-registry secret names as an array                                                     | `[]` (does not add image pull secrets to deployed pods)      |
 | `metrics.podAnnotations`                  | Additional annotations for Metrics exporter                                                          | `{prometheus.io/scrape: "true", prometheus.io/port: "9112"}` |
+| `additionalRepos.enabled`                 | Enables the use of additional repos in the default Airflow Workers                                   | `false` 							  |
+| `additionalRepos.repos[o].name`	    | Descriptive name of a repo to sync			 					   | `nil`							  |
+| `additionalRepos.repos[0].repository`	    | Link to the additional repository to sync 							   | `nil` 							  |
+| `additionalRepos.repos[0].branch` 	    | Branch from repository to sync									   | `nil`							  | 
+| `additionalRepos.repos[0].interval`	    | Interval to pull the repository									   | 60 							  |
+| `additionalRepos.repos[0].sshKeySecret`   | Kubernetes secret containing SSH keys for this repo						   | `nil`							  |
+| `workers[0].name`	      		    | Descriptive name of a worker to add								   | `nil`							  |
+| `workers[0].volumes[0].name`		    | Name of volume to add to this worker 								   | `nil`							  | 
+| `workers[0].image.registry`    	    | Registry for this workers image									   | `nil`							  |
+| `workers[0].image.repository`		    | Name of this worker image										   | `nil`							  | 
+| `workers[0].image.tag`		    | Tag for this worker image										   | `nil`							  | 
+| `workers[0].image.pullPolicy` 	    | Pull policy for this worker image									   | `nil`							  | 
+| `workers[0].additionalRepos.enabled`	    | Enables the use of additional repos for this worker						   | `false`							  |
+| `workers[0].additionalRepos.repos[0].name` | Descriptive name of a repo to sync with this worker 	 					   | `nil`							  |
+| `workers[0].additionalRepos.repos[0].repository` | Link to the additional repository to sync with this worker					   | `nil` 							  |
+| `workers[0].additionalRepos.repos[0].branch` | Branch from repository to sync with this worker							   | `nil`							  | 
+| `workers[0].additionalRepos.repos[0].interval` | Interval to pull the repository for this worker							   | 60 							  |
+| `workers[0].additionalRepos.repos[0].sshKeySecret` | Kubernetes secret containing SSH keys for this repo						   | `nil`							  |
```

Comment:
I would name this parameter `workers[0].additionalRepos.repos[0].sshKeysSecretName`